packages/terra-site/webpack.prod.config.js

```diff
@@ -16,9 +17,13 @@ config.plugins.push(new webpack.DefinePlugin({
 // Clean build before running
 config.plugins.push(new CleanPlugin('build', { exclude: ['stats.json'] }));
 
+// Minify css and js
+config.plugins.push(new webpack.LoaderOptionsPlugin({ minimize: true }));
+
```

Comment:
To explain tree-shaking: 
> Javascript bundling will only include code that is necessary to run your application. It has been made available by the static nature of ES2015 modules (exports and imports can’t be modified at runtime), which lets us detect unused code at bundle time. refer to [this blog](https://blog.craftlab.hu/how-to-do-proper-tree-shaking-in-webpack-2-e27852af8b21)

To add UglifyJsPlugin or bablili in thie PR, I have three concerns: 
1. I'm not sure their pros and cons. We may need to create another PR to document those concerns. It helps us to track the decision and convince consumers with our final choice. 
2. And in this PR, we add `cheap-source-map` into the webpack config. We may need to test that with minified files by UglifyJsPlugin or bablili.
3. As this will be our recommendation across all consumers, we may want to compare the compression ratio to achieve the best performance.